Rakefile

```diff
@@ -10,8 +10,7 @@ begin
   require 'github_changelog_generator/task'
   GitHubChangelogGenerator::RakeTask.new :changelog do |config|
     config.future_release = FacterDB::Version::STRING
-    config.include_labels = %w[enhancement bug]
-    config.exclude_labels = %w[duplicate question invalid wontfix maintenance github_actions]
+    config.exclude_labels = %w[duplicate question invalid wontfix wont-fix maintenance skip-changelog github_actions]
```

Comment:
Maybe we should add the `dependencies` label to this list, to filter dependabot PRs in the changelog (no much value for end-users)?

Reply:
We discussed it on Slack yesterday and @bastelfreak thought we should have them

Reply:
Thinking about the changelog format, I wonder if it makes sense for the future to separate the entries in the changelog by whether they are:
- Fact generation tooling
- Factset changes
- Gem usage changes

I wouldn't change it now, but since there is a v3 on the horizon, maybe that would make sense.

Reply:
we've the same config in other gems. We only exclude github_actions PRs because the rest could have impact on users.